saleor/dashboard/order/forms.py

```diff
@@ -110,25 +110,25 @@ def release(self):
         return True
 
 
-class MoveItemsForm(forms.Form):
+class MoveLinesForm(forms.Form):
     NEW_SHIPMENT = 'new'
     quantity = QuantityField(
-        label=pgettext_lazy('Move items form label', 'Quantity'),
+        label=pgettext_lazy('Move lines form label', 'Quantity'),
         validators=[MinValueValidator(1)])
     target_group = forms.ChoiceField(
-        label=pgettext_lazy('Move items form label', 'Target shipment'))
+        label=pgettext_lazy('Move lines form label', 'Target shipment'))
 
     def __init__(self, *args, **kwargs):
-        self.item = kwargs.pop('item')
-        super(MoveItemsForm, self).__init__(*args, **kwargs)
+        self.line = kwargs.pop('item')
```

Comment:
If you renamed `item` to `line`, shouldn't you rename variable passed as well?

Reply:
Missed this one, fixed, thank you!